.github/workflows/build-project.yaml

```diff
@@ -77,9 +77,13 @@ jobs:
           if (( ${+RUNNER_DEBUG} )) setopt XTRACE
 
           print '::group::Clean Homebrew Environment'
-          typeset -a to_remove=()
+          local -a to_remove=()
 
-          if (( #to_remove > 0 )) brew uninstall --ignore-dependencies ${to_remove}
+          for formula (curl) {
+            if [[ -d ${HOMEBREW_PREFIX}/opt/${formula} ]] to_remove+=(${formula})
```

Comment:
Is there a specific reason why `curl` is marked for removal in this PR?

Reply:
I was just blindly copying the code from:
https://github.com/obsproject/obs-studio/blob/c51c1175fb3c565b1f0d1a04bcff87551000be87/.github/workflows/build-project.yaml#L94-L96

I am fine removing.